modules/oadp-release-notes-1-2-2.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-2-2_{context}"]
+= OADP 1.2.2 release notes
+
+
+[id="new-features-1-2-2_{context}"]
+== New features
+
+There are no new features in the release of {oadp-first} 1.2.2.
+
+// :FeatureName: OADP Data Mover
+// include::snippets/technology-preview.adoc[]
+
+[id="resolved-issues-1-2-2_{context}"]
+== Resolved issues
+
+The following highlighted issues have been resolved in OADP 1.2.2:
+
+
+.Restic restore partially failed due to Pod Security standard
+
+In previous releases of OADP 1.2, {ocp} 4.14 enforced a pod security admission (PSA) policy that hindered the readiness of pods during a Restic restore process.
+
+This issue has been resolved in the OADP 1.1.6 and OADP 1.2.2 releases. Therefore, it is recommended that users upgrade to these releases.
+
+For more information, see link:https://docs.openshift.com/container-platform/4.14/backup_and_restore/application_backup_and_restore/troubleshooting.html#oadp-restic-restore-failing-psa-policy_oadp-troubleshooting[Restic restore partially failing on OCP 4.14 due to changed PSA policy]. link:https://issues.redhat.com/browse/OADP-2094[(OADP-2094)]
+
+
+.Backup of an app with internal images partially failed with plugin panicked error
+
+In previous releases of OADP 1.2, the backup of an application with internal images partially failed with plugin panicked error returned. The backup partially fails with this error in the velero logs:
+
+[source,terminal]
+----
+time="2022-11-23T15:40:46Z" level=info msg="1 errors encountered backup up item" backup=openshift-adp/django-persistent-67a5b83d-6b44-11ed-9cba-902e163f806c logSource="/remote-source/velero/app/pkg/backup/backup.go:413" name=django-psql-persistent
+time="2022-11-23T15:40:46Z" level=error msg="Error backing up item" backup=openshift-adp/django-persistent-67a5b83d-6b44-11ed-9cba-902e163f8
+----
+
+This issue has been resolved in OADP 1.2.2. link:https://issues.redhat.com/browse/OADP-1057[(OADP-1057)].
+
+
+.ACM cluster restore was not functioning as expected due to restore order
+
+In previous releases of OADP 1.2, ACM cluster restore was not functioning as expected due to restore order. ACM applications were removed and re-created on managed cluster(s) after restore activation. link:https://issues.redhat.com/browse/OADP-2505[(OADP-2505)]
+
+
+.KubeVirt VM's using filesystemOverhead failed when backing up and restoring due to volume size mismatch
+
+In previous releases of OADP 1.2, KubeVirt VM's using `filesystemOverhead` failed when backing up and restoring because of a volume size mismatch. OADP was not considering the additional size of the persistent volume (PV) with the `filesystemOverhead` setting.  The backup and restore cloned PV's failed to bind, as a result, the backup and restore failed. link:https://issues.redhat.com/browse/OADP-2144[(OADP-2144)]
```

Comment:
I think we should change the description here (or even the issue title)
This issue might/probably not related to kubevirt fs-overhead
even if it is, Users might bumped into this issue outside of kubevirt.
@shubham-pampattiwar  wdyt?

Reply:
@amastbau yes I agree with you, we can add something more generic like, (due to storage provider implementation choices) whenever there is a difference between the application PVC storage request and the snapshot size of the same pvc this issue will occur and OADP 1.2 datamover is able to handle that now after this fix.